app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
```

Comment:
Add a blank line between different groups of imports to follow PEP-8 in all files where applicable